lib/bump.rb

```
@@ -155,6 +157,21 @@ def bump(file, current, next_version, options)
         ["Bump version #{current} to #{next_version}", 0]
       end
 
+      def open_changelog options, log
+        if options[:editor]
+          return ["Could not find '#{options[:editor]}'", 1] unless command_exists?(options[:editor])
```

I'd say just start it and let it crash 🤷‍♂